kitchen.dokken.yml

```diff
@@ -37,6 +40,17 @@ platforms:
       image: dokken/centos-7
       pid_one_command: /usr/lib/systemd/systemd
 
+  - name: centos-7-chef-13
+    driver:
+      image: dokken/centos-7
+      pid_one_command: /usr/lib/systemd/systemd
+      chef_version: 13
```

Comment:
Why Chef 13? Really we should support 15 and 16 by default only

Reply:
Because the current cookbook still supports 13 and it works. We don't have any hard to reason to bump this up to 15/16 (yet). We'll likely do that when I convert this over to a resource/library cookbook.